Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PipelineBrush/PipelineBrushXaml.bind

```diff
@@ -20,29 +20,14 @@
               Grid.Column="2"
               Height="400">
         <Border.Background>
-          <brushes:PipelineBrush>
-            <brushes:PipelineBrush.Effects>
-              <effects:BackdropEffect Source="Backdrop"/>
-              <effects:LuminanceToAlphaEffect/>
-              <effects:OpacityEffect Value="0.4"/>
-              <effects:BlendEffect Mode="Multiply">
-                <effects:BlendEffect.Input>
-                  <effects:BackdropEffect Source="Backdrop"/>
-                </effects:BlendEffect.Input>
-              </effects:BlendEffect>
-              <effects:BlurEffect Value="16"/>
-              <effects:ShadeEffect Color="#FF222222" Intensity="0.2"/>
-              <effects:BlendEffect Mode="Overlay" Placement="Background">
-                <effects:BlendEffect.Input>
-                  <effects:TileEffect Uri="ms-appx:///Assets/BrushAssets/NoiseTexture.png"/>
-                </effects:BlendEffect.Input>
-              </effects:BlendEffect>
-              <effects:BlendEffect Mode="Overlay" Placement="Background">
-                <effects:BlendEffect.Input>
-                  <effects:ImageEffect Uri="ms-appx:///SamplePages/DropShadowPanel/Unicorn.png"/>
-                </effects:BlendEffect.Input>
-              </effects:BlendEffect>
-            </brushes:PipelineBrush.Effects>
+          <brushes:PipelineBrush Source="{effects:BackdropSource}">
```

Comment:
We can remove the BackdropSource as that's the default now, but should we maybe put another source for one of them?

Also, I was trying to modify the Blur Amount value below, but I didn't see much effect...?

Reply:
I kinda left it there just to show the property to users. I don't think we can change the second source with the current pipeline, as right now that luminosity effect only makes sense if it's applied to the same source, as it's modifying the luminosity of the same pixels that are sampled from the secondary pipeline. If the pipeline being blended in used a difference source, the effect would basically just end up creating weird artifacts instead 🤔